app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -348,4 +351,11 @@ class UserSession @Inject constructor(
   fun accessToken(): Optional<String> {
     return accessTokenPreference.get()
   }
+
+  fun startForgotPinFlow(retryCount: Int = 0): Completable {
```

Comment:
Hrm, should we make this function name more explicit to what it does?

Reply:
TODO: Think of a better name when the feature is finished.